binding/HarfBuzzSharp.Classic/HarfBuzzSharp.Classic.csproj

```diff
@@ -41,8 +41,10 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetFramework)' == 'tizen40'">
     <!-- Tizen -->
-    <None Include="..\..\output\native\tizen\armel\libHarfBuzzSharp.*" Link="nuget\build\$(TargetFramework)\arm\%(Filename)%(Extension)" />
-    <None Include="..\..\output\native\tizen\i386\libHarfBuzzSharp.*" Link="nuget\build\$(TargetFramework)\x86\%(Filename)%(Extension)" />
+    <None Include="..\..\output\native\tizen\armel\libHarfBuzzSharp.*" Link="nuget\runtimes\tizen-armel\%(Filename)%(Extension)" />
+    <None Include="..\..\output\native\tizen\arm64\libHarfBuzzSharp.*" Link="nuget\runtimes\tizen-arm64\%(Filename)%(Extension)" />
+    <None Include="..\..\output\native\tizen\i386\libHarfBuzzSharp.*" Link="nuget\runtimes\tizen-x86\%(Filename)%(Extension)" />
+    <None Include="..\..\output\native\tizen\i386\libHarfBuzzSharp.*" Link="nuget\runtimes\linux-x86\%(Filename)%(Extension)" />
```

Comment:
Do we still need the `linux-x86` runtime? I am not sure we can do this as it will replace the linux binary - and will also cause the tizen binary to run on a linux (desktop) machine.

What we can do is actually just keep the `tizen-x86` version, and pack it twice in the tizen nuget.